src/components/Movies.js

```diff
@@ -1,9 +1,22 @@
 import React from 'react';
 import Poster from './Poster';
+import scrollLeftButton from './scroll-left.png';
+import scrollRightButton from './scroll-right.png';
 import './Movies.css';
 
 const Movies = ({movies}) => {
 
+  const scroller = React.useRef();
+
+  const scrollLeft = () => {
+    console.log(scroller.current);
```

Comment:
I sure we wont miss it later to take this out but just making a note that there was a console.log left in! (Actually glad you left it so I could see more about what you added!)